### DISCUSSION TITLE
Time left

### DISCUSSION THREAD

**Discussion by dereineLauch** [cpiber/NebulaEnhance#9] [Ideas]
I would like to know how long the video will last. 
By clicking on the time-thing it will change from 'how long the video has been running' to 'how long the video will last'.
Like that:
![image](https://user-images.githubusercontent.com/84781281/133276113-73127327-4b48-4866-822c-5bb8fbd150b6.png)

The idea comes from the YouTube App:
![image](https://user-images.githubusercontent.com/84781281/133291121-78b51a87-83ac-48a0-8a39-4142eebf127b.png)

**Comment by cpiber** [comment]
I looked into it a bit, I think it should be doable, I'll start working on it tomorrow

**Reply by cpiber** [reply]
It actually wasn't too difficult to do, could you please check it? 
[enhancer_for_nebula-1.0.02.zip](https://github.com/cpiber/NebulaEnhance/files/7172304/enhancer_for_nebula-1.0.02.zip)

To install:
- in Chrome, please go to Setting, click Extensions (lower left), enable Developer Mode (upper right) and select "Load unpacked". Here select the unzipped folder (should contain folders _locale, icons, scripts, styles).
- in Firefox, please go to about:addons, click on the gear (top right), select Debug addons and click "Load Temporary Addon". Here select the zip you downloaded.

**Reply by dereineLauch** [reply]
Thank you. It works fine.
Sorry for the late answer.
